apis/upbound/mcp-connector/composition.yaml

```diff
@@ -0,0 +1,161 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: xconnectors.mcp.caas.upbound.io
+spec:
+  writeConnectionSecretsToNamespace: upbound-system
+  compositeTypeRef:
+    apiVersion: mcp.caas.upbound.io/v1alpha1
+    kind: XConnector
+  resources:
+    - name: mcp-serviceaccount
+      base:
+        apiVersion: iam.upbound.io/v1alpha1
+        kind: Robot
+      patches:
+        - fromFieldPath: spec.parameters.deletionPolicy
+          toFieldPath: spec.deletionPolicy
+        - fromFieldPath: spec.parameters.organizationName
+          toFieldPath: spec.forProvider.owner.name
+        - fromFieldPath: metadata.labels[crossplane.io/claim-name]
```

Comment:
looks like a strong dependency on Claim invocation. Is this resource explicitly designed to be invoked as Claim only? Otherwise, it will break as XR. It's probably by design so just double checking.

Reply:
Yes in upbound use the claim